### PR TITLE
Remove default for hive.cache.location

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixConfig.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ public void testDefaults()
         assertRecordedDefaults(recordDefaults(RubixConfig.class)
                 .setBookKeeperServerPort(CacheConfig.DEFAULT_BOOKKEEPER_SERVER_PORT)
                 .setDataTransferServerPort(CacheConfig.DEFAULT_DATA_TRANSFER_SERVER_PORT)
-                .setCacheLocation("/tmp")
+                .setCacheLocation(null)
```
Comment (martint) [inline presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixConfig.java:34]: This looks strange. The field is `@NotNull`, but we're recording a null.

Comment (electrum) [reply presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixConfig.java:34]: This how you do a required config (something the user is forced to set).

**Comment by electrum** [comment]
Might be good to change the test from `/etc` to `/cache` or something (since `/etc` both looks real and is not something you'd use).

File: presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixConfig.java (hunk 1 above)
Comment (martint) [reply presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixConfig.java:34]: I know... it just looks weird and we should figure out a better way to represent that at some point.

**Pull request merged**